api-report/tree.api.md

```diff
@@ -32,7 +33,25 @@ export abstract class BrandedType<ValueType, Name extends string> {
 }
 
 // @public (undocumented)
-export function buildForest(): IEditableForest;
+export function buildForest(schema: StoredSchemaRepository): IEditableForest;
+
+// @public (undocumented)
+export abstract class ChangeEncoder<TChange> {
+    decodeBinary(formatVersion: number, change: IsoBuffer): TChange;
+    // (undocumented)
+    abstract decodeJson(formatVersion: number, change: JsonCompatibleReadOnly): TChange;
+    encodeBinary(formatVersion: number, change: TChange): IsoBuffer;
+    // (undocumented)
+    abstract encodeForJson(formatVersion: number, change: TChange): JsonCompatibleReadOnly;
+}
+
+// @public
+export interface ChangeHandler<TChange> {
+    // (undocumented)
+    readonly encoder: ChangeEncoder<TChange>;
```

Comment:
Document (and below)

Reply:
There are quite a few more undocumented things added so I'll let you go through those instead of spamming comments :)

Reply:
I think I got everything.